### PR TITLE
Add-on options can be null

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

**Review by bramkragten — APPROVED** [review]

File: worker/src/utils/validate.ts [hunk 1]
```diff
@@ -30,9 +30,9 @@ export const IncomingPayload = object({
     array(
       object({
         slug: string(),
-        protected: boolean(),
+        protected: nullable(boolean()),
```
Comment (bramkragten) [inline worker/src/utils/validate.ts:33]: Should we default it to the default value then?

Comment (ludeeus) [reply worker/src/utils/validate.ts:33]: I looked at the supervisor source, `None` is the default

Comment (bramkragten) [reply worker/src/utils/validate.ts:33]: But that would be the same as `True` here right?

File: worker/src/utils/validate.ts [hunk 2]
```diff
@@ -30,9 +30,9 @@ export const IncomingPayload = object({
...
         version: optional(nullable(string())),
-        auto_update: boolean(),
+        auto_update: nullable(boolean()),
```
Comment (bramkragten) [inline worker/src/utils/validate.ts:35]: And for this one `False`?

File: worker/src/utils/validate.ts (hunk 1 above)
Comment (bramkragten) [reply worker/src/utils/validate.ts:33]: OK, so use `create` instead of `assert`? 😄

**Pull request closed without merging**